lib/pansophy/local/create_file.rb

```diff
@@ -0,0 +1,28 @@
+module Pansophy
+  module Local
+    class CreateFile
+      include Adamantium::Flat
+
+      def initialize(path, body)
+        @pathname = Pathname.new(path)
+        @body     = body
+      end
+
+      def call(options = {})
+        prevent_overwrite! unless options[:overwrite]
+        @pathname.dirname.mkpath
+        ::File.open(@pathname, 'w') do |f|
```

Comment:
Unfortunately @adz, I had to use the old syntax for pre Ruby 2 (or 2.2, can't remember) compatibility.

On Tue, Apr 21, 2015 at 8:55 PM, Adam Davies <notifications@github.com>
wrote:

>> @@ -0,0 +1,28 @@
>> +module Pansophy
>> +  module Local
>> +    class CreateFile
>> +      include Adamantium::Flat
>> +
>> +      def initialize(path, body)
>> +        @pathname = Pathname.new(path)
>> +        @body     = body
>> +      end
>> +
>> +      def call(options = {})
>> +        prevent_overwrite! unless options[:overwrite]
>> +        @pathname.dirname.mkpath
>> +        ::File.open(@pathname, 'w') do |f|
> @pathname.write(@body)
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/sealink/pansophy/pull/3/files#r28769174